armi/reactor/components/componentParameters.py

```diff
@@ -165,6 +165,18 @@ def getHoledHexagonParameterDefinitions():
     return pDefs
 
 
+def getHexHoledCircleParameterDefinitions():
+    """Return parameters for HexHoledCircle."""
+
+    pDefs = parameters.ParameterDefinitionCollection()
+    with pDefs.createBuilder(location=ParamLocation.AVERAGE, saveToDB=True) as pb:
+        pb.defParam("holeOP", units="cm", description="Pitch of interior hole")
+
+        pb.defParam("od", units="cm", description="Outer diameter")
```

Comment:
Okay, apparently this line needs to be removed for your tests to pass: https://github.com/terrapower/armi/actions/runs/4145259916/jobs/7170527291#step:7:445

Sorry about that, cause it was me who recommended it in the first place :\